Validation/GlobalDigis/src/GlobalDigisAnalyzer.cc

```diff
@@ -1468,7 +1468,7 @@ void GlobalDigisAnalyzer::fillMuon(const edm::Event &iEvent, const edm::EventSet
   bool validrpcdigi = true;
   if (!rpcDigis.isValid()) {
     LogDebug(MsgLoggerCat) << "Unable to find rpcDigis in event!";
-    validrpcdigi = false;
+    [[clang::suppress]] validrpcdigi = false;
```

Comment:
clang analyzer complains as we force set ` validrpcdigi = false;` at few line later